ui-admin/src/login/IdleStatusMonitor.tsx

```diff
@@ -118,7 +118,7 @@ const InactivityTimer = ({ maxIdleSessionDuration, idleWarningDuration, doSignOu
 
   useEffect(() => {
     const targetEvents = ['click', 'keydown']
-    const updateLastActive = () => lastRecordedActivity.current = Date.now()
+    const updateLastActive = () => { lastRecordedActivity.current = Date.now() }
```

Comment:
:( I wish the eslint check didn't require the extra ceremony around inline functions like this... especially in cases like this where typescript is saying that the given function returns `void` anyway.